homework1.js

```diff
@@ -0,0 +1,143 @@
+// Application definition
+
+/**
+ * @returns random element from any array
+ */
+Array.prototype.random = function () {
+  return this[Math.floor(Math.random() * this.length)];
+};
+
+/**
+ *
+ * @param {string} engine engine type (ex: 'v8')
+ * @param {string} color car color (ex: 'red')
+ * @param {string} carBrand car brand (ex: 'bmw')
+ * @param {integer} manufacturingYear car manufacturing year (ex: 1993)
+ * @param {boolean} fammilyCar familyCar (ex: true) 
+ */
+function Car(engine, color, carBrand, manufacturingYear, familyCar) {
+  this.engine = engine;
+  this.color = color;
+  this.carBrand = carBrand;
+  this.manufacturingYear = manufacturingYear;
+  this.familyCar = familyCar;
+}
+
+Car.prototype.startEngine = function () {
+  console.log('vrum vrum');
+}
+
+Car.prototype.stopEngine = function () {
+  console.log('no more vrum vrum');
+}
+
+Car.prototype.paintCar = function (newColour) {
+  return 'The colour of ' + this.Car.carBrand + ' has been changed from ' + this.carBrand.color + ' to ' + newColour;
```

Comment:
As putea sa iti recomand stilul asta pentru template:
```suggestion
`The colour of ${this.Car.carBrand}  has been changed from ${this.carBrand.color}  to ${newColour}`;
```

Reply:
O sa-l folosesc de acum incolo, am facut update si in tema, multumesc